src/hostapi/asio/pa_asio.cpp

```diff
@@ -1304,9 +1304,10 @@ PaError PaAsio_Initialize( PaUtilHostApiRepresentation **hostApi, PaHostApiIndex
                 continue;
             }
 
-            // Blacklist dynamic loading of JackRouter in 32-bit applications.
+            // Blacklist dynamic loading of JackRouter ASIO driver by 64-bit applications.
             // This causes an instant crash due to a long-standing address translation error.
             //   https://github.com/jackaudio/jack2/issues/332
+            //   https://github.com/jackaudio/jack2/issues/275
```

Comment:
This is tangential to this PR, but I got PortAudio to build on Windows with the JACK host API: https://github.com/PortAudio/portaudio/pull/618 Considering the ASIO backend explicitly prevents using JackRouter, I think this would be the only way for PortAudio applications to use JACK on Windows.

It builds but it doesn't seem to work on Windows. I have not had a chance to debug it yet. Help would be appreciated if you are interested.